<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficCounterMainWindow</class>
 <widget class="QMainWindow" name="TrafficCounterMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrafficCounter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="videoFrameDisplayLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FileVideoSourceProgressBar" name="playbackProgressProgressBar">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openVideoAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuPlayback_mode">
     <property name="title">
      <string>Playback mode</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="originalFrameAction"/>
     <addaction name="processedFrameAction"/>
    </widget>
    <addaction name="sourceSettingsAction"/>
    <addaction name="playbackActionsAction"/>
    <addaction name="menuPlayback_mode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="databaseSettingsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="openVideoAction">
   <property name="text">
    <string>&amp;Open video</string>
   </property>
  </action>
  <action name="sourceSettingsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="playbackActionsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Playback actions</string>
   </property>
  </action>
  <action name="databaseSettingsAction">
   <property name="text">
    <string>&amp;Database</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="originalFrameAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Original frame</string>
   </property>
  </action>
  <action name="processedFrameAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Processed frame</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileVideoSourceProgressBar</class>
   <extends>QProgressBar</extends>
   <header>src/widgets/FileVideoSourceProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
